doc/userguide/rules/transforms.rst

```diff
@@ -188,3 +188,53 @@ Example::
 
     alert http any any -> any any (msg:"HTTP ua only"; http.header_names; \
        bsize:16; content:"|0d 0a|User-Agent|0d 0a 0d 0a|"; nocase; sid:1;)
+
+.. _from_base64:
+
+from_base64
```

Comment:
can this explain what happens if data is not base64, or partly invalid? Will buffer be empty? Will invalid bytes be put in the buffer?

Reply:
I'll add that to the description of the transform.